src/main/server.ts

```diff
@@ -1,26 +1,64 @@
 #!/usr/bin/env node
 
-import { app } from './app'
+import * as tsConfigPaths from 'tsconfig-paths'
+import { cpus } from 'os'
+import * as express from 'express'
 import * as fs from 'fs'
 import * as path from 'path'
 import * as https from 'https'
+import * as cluster from 'cluster'
 
+const tsConfig = require('../../tsconfig.json')
+const numberOfCores = cpus().length
```

Comment:
That should be more in local scope of `if (cluster.isMaster) {` block to keep declaration as close as possible to the place it is used.